src/tests/integration/test_generic_db_operations.py

```diff
@@ -391,34 +393,33 @@ def test_sqlplot_boxplot(ip_with_dynamic_db, cell, request, test_table_name_dict
         ("ip_with_mariaDB"),
         ("ip_with_SQLite"),
         ("ip_with_duckDB"),
-        ("ip_with_duckDB_native"),
-        ("ip_with_MSSQL"),
+        pytest.param(
+            "ip_with_duckDB_native",
+            marks=pytest.mark.xfail(reason="not supported yet for native connections"),
+        ),
+        pytest.param(
+            "ip_with_MSSQL",
+            marks=pytest.mark.xfail(reason="not working yet"),
+        ),
         ("ip_with_Snowflake"),
         ("ip_with_oracle"),
     ],
 )
-def test_sql_cmd_magic_uno(ip_with_dynamic_db, request, capsys):
+def test_sql_cmd_magic_uno(ip_with_dynamic_db, request, test_table_name_dict):
     ip_with_dynamic_db = request.getfixturevalue(ip_with_dynamic_db)
+    table = test_table_name_dict["numbers"]
 
-    ip_with_dynamic_db.run_cell(
-        """
-    %%sql sqlite://
-    CREATE TABLE test_numbers (value);
-    INSERT INTO test_numbers VALUES (0);
-    INSERT INTO test_numbers VALUES (4);
-    INSERT INTO test_numbers VALUES (5);
-    INSERT INTO test_numbers VALUES (6);
-    """
-    )
-
-    ip_with_dynamic_db.run_cell(
-        "%sqlcmd test --table test_numbers --column value" " --less-than 5 --greater 1"
-    )
+    ip_with_dynamic_db.run_cell(f"%sql select * from {table}")
 
-    _out = capsys.readouterr()
+    with pytest.raises(UsageError) as excinfo:
+        ip_with_dynamic_db.run_cell(
+            f"%sqlcmd test --table {table} --column numbers_elements "
+            "--less-than 1 --greater 2"
+        )
 
-    assert "less_than" in _out.out
-    assert "greater" in _out.out
+    assert "The above values do not not match your test requirements." in str(
```

Comment:
not not  -> not

This comes from [here](https://github.com/ploomber/jupysql/blob/dev/refactor-integration/src/sql/cmd/test.py#L176)

Reply:
fixed